dtcli/scripts/dt.py

```diff
@@ -180,6 +181,14 @@ def extension_dev():
     pass
 
 
+@main.group(aliases=["utilities"], hidden=True)
```

Comment:
Please use Typer
The alias is a nice idea, though it won't be needed here, since users of this group are mostly scripts